1984bot.py

```diff
@@ -52,7 +52,79 @@
 @bot.event
 async def on_ready():
     print('ONLINE')
-
+    
+async def on_message(self, message):
+    if message.author == client.user:
+       return
+     string = message.content
+     stringArr = ['place', 'triggers', 'here']
+     #All below replaces characters in a string (common substitutions) to prevent people from escaping the blacklist
+     string = re.sub('1', 'i', string)

```

Comment:
I'll probably pretty these up by shoving them into two arrays

Reply:
Two arrays would allow for different blacklist priorities, I like this idea.